### DISCUSSION TITLE
Season shows partially available when it is fully available.

### DISCUSSION THREAD

**Discussion by Bubbgump209** [seerr-team/seerr#289] [Q&A]
I am running Jellyseer 1.3.0. Under various shows it shows "partially available" for certain seasons when the complete season is there. Where is Jellyseerr pulling its data to make this determination as Sonarr shows that the season is complete - so I would expect Jellyseerr to show the same. It feels very similar to this issue in Overseerr https://github.com/sct/overseerr/issues/723 as these are shows that are like "Showname - S01E01E02E03 - 1080p.mkv".

**Comment by fallenbagel** [comment]
Does the same happen in 1.2.0?

**Reply by Bubbgump209** [reply]
I pulled 1.2.1 and 1.2.0 - both are actually worse as they show even more seasons as partially available. FWIW I ran a manual scan after I pulled each.

**Reply by fallenbagel** [reply]
Did you try it like after cleaeing the data for that show?

**Reply by Bubbgump209** [reply]
I just redid everything with both 1.2.0 and 1.2.1. Choose the show > gear > Clear data and then ran a manual scan. It was the same behavior as before where it was worse and all seasons showed as "partially available".

**Reply by fallenbagel** [reply]
> I just redid everything with both 1.2.0 and 1.2.1. Choose the show > gear > Clear data and then ran a manual scan. It was the same behavior as before where it was worse and all seasons showed as "partially available".

Okay so for now I suggest turning on availability on sonarr and radarr too. And use 1.3.0 or develop. My guess is its having trouble reading either multi episodes or messing up due to specials. Not exactly sure but I also have one series like that (I manually made it available instead)

Will have to check what's happening. My assumption is it would prolly work well once individual episode request feature is added

**Reply by fallenbagel** [reply]
Could you show me a screenshot of one series from jellyseerr and jellyfin both episode list to see what's happening. (Would love it if you could come to discord so it's easier to debug)

**Reply by fallenbagel** [reply]
So I made some progress. So the show I had partially available, what I did was I removed the `.nfo` files and then cleared the data for it on jellyseerr then rescanned it in jellyfin which added new metadata and now it's working and shown as fully available.

A simple replace metadata scan might work too

**Comment by chechu10** [comment]
That's because If I'm not wrong, metadata info in jellyseer come from TheMovieDB and you can't customize it. Then sometimes is a bit different from TheTVDB, what is maybe using on your Jellyfin priorizing over TheMovieDB (TV).

Then, sometimes, in some TV Shows, organization is little different, in TheTVDB maybe have 15 programs in a season and one special, while in TheMovieDB is 16 programs in this season.

Can you validate your program with these two platforms and tell us the differences, please?

Regards.

**Reply by fallenbagel** [reply]
We solved it on discord. Turns out sometimes when you have specials appear in the seasons itself, jellyseerr has trouble reading the full season and throws it as partially available even when all episodes is available. When they are configured to not display inside seasons, it works smoothly.

This issue should be fixed once availability per episode sync is introduced since then it'd look for the id of the episode instead of a full season and all those specials would be ignored
